config/config.go

```diff
@@ -91,13 +92,15 @@ func setDefaults() {
 	viper.SetDefault("blockstore.s3.region", DefaultBlockStoreS3Region)
 	viper.SetDefault("blockstore.s3.streaming_chunk_size", DefaultBlockStoreS3StreamingChunkSize)
 	viper.SetDefault("blockstore.s3.streaming_chunk_timeout", DefaultBlockStoreS3StreamingChunkTimeout)
+	viper.SetDefault("blockstore.s3.max_retries", DefaultS3MaxRetries)
 
 	viper.SetDefault("gateways.s3.domain_name", DefaultS3GatewayDomainName)
 	viper.SetDefault("gateways.s3.region", DefaultS3GatewayRegion)
 
 	viper.SetDefault("blockstore.gs.s3_endpoint", DefaultBlockStoreGSS3Endpoint)
 	viper.SetDefault("blockstore.gs.streaming_chunk_size", DefaultBlockStoreGSStreamingChunkSize)
 	viper.SetDefault("blockstore.gs.streaming_chunk_timeout", DefaultBlockStoreGSStreamingChunkTimeout)
+	viper.SetDefault("blockstore.gs.max_retries", DefaultS3MaxRetries)
```

Comment:
can remove this one - google client already holds a retry mechanism

Reply:
Done, thanks!
But I amended this commit and rewrote history.  Otherwise it's  just noise.